src/definitions/ping360.json

```diff
@@ -82,12 +105,14 @@
                     {
                         "name": "bootloader",
                         "type": "u8",
-                        "description": "0 = skip bootloader; 1 = run bootloader"
+                        "description": "0 = skip bootloader; 1 = run bootloader",
+                        "default": "0"
```

Comment:
minimum and maximum are missing here.

Reply:
This is not a range value. It is an enumeration. I think range does not make sense here.

Reply:
if it's an enum, it's better to use identifiers instead of integers in the json